### ISSUE TITLE
Allow to disable the tray icon on platforms that don't know the concept

### ISSUE THREAD

**Issue by SISheogorath** [lensapp/lens#2213]
**What would you like to be added**:
Ability to disable the tray icon and with that the run in background functionality. By either using a CLI flag or a setting in preferences. (Automatically would be even greater but is, as far as I know, not possible in a meaningful way in electron.)

**Why is this needed**:
The desktop environment of GNOME 3 removed the concept of tray icons. Therefore a tray icon can never show up.

**Environment you are Lens application on:**

- Kubernetes distribution: any
- Desktop OS: Fedora Linux 33

**Comment by Nokel81** [comment]
For some more information, if the Tray icon doesn't show up. What does Lens do when you close the window? (Since iirc on linux there isn't normally process persistence when the UI closes).

**Comment by SISheogorath** [comment]
Currently it remains running, since electron successfully registers the tray icon, even when it never shows up, resulting in lens running in background with no indicator for that.

**Comment by defenestration** [comment]
I would also like the option to be able to just disable the tray icon, even for OS that do support it (macos here).

**Comment by dparker18** [comment]
> I would also like the option to be able to just disable the tray icon, even for OS that do support it (macos here).

Agreed. For me the forced tray icon when closing the app is clutter and mildly obnoxious.
